kinto/tests/test_main.py

```diff
@@ -132,3 +133,40 @@ def test_cli_start_with_reload_runs_pserve_with_reload(self):
             assert res == 0
             assert mocked_pserve.call_count == 1
             assert '--reload' in mocked_pserve.call_args[0][0]
+
+    def test_cli_can_display_kinto_version(self):
+        with mock.patch('sys.stderr', new_callable=StringIO) as mock_stderr:
+            res = main(['--version'])
+            assert mock_stderr.getvalue() == '%s\n' % kinto_version
+            assert res == 0
+
+    def test_cli_can_configure_logger_in_verbose(self):
+        with mock.patch('kinto.__main__.logging') as mocked_logging:
+            main(['-v', '--ini', TEMP_KINTO_INI, 'init',
+                        '--backend', 'memory'])
+            mocked_logging.basicConfig.assert_called_with(
+                level=mocked_logging.INFO,
+                format=DEFAULT_LOG_FORMAT)
+
+    def test_cli_can_configure_logger_in_quiet(self):
+        with mock.patch('kinto.__main__.logging') as mocked_logging:
+            main(['-q', '--ini', TEMP_KINTO_INI, 'init',
+                        '--backend', 'memory'])
+            mocked_logging.basicConfig.assert_called_with(
+                level=mocked_logging.CRITICAL,
+                format=DEFAULT_LOG_FORMAT)
+
+    def test_cli_can_configure_logger_in_debug(self):
+        with mock.patch('kinto.__main__.logging') as mocked_logging:
+            main(['-D', '--ini', TEMP_KINTO_INI, 'init',
+                        '--backend', 'memory'])
+            mocked_logging.basicConfig.assert_called_with(
+                level=mocked_logging.DEBUG,
+                format=DEFAULT_LOG_FORMAT)
+
+    def test_cli_use_default_logging_logger(self):
+        with mock.patch('kinto.__main__.logging') as mocked_logging:
+            main(['--ini', TEMP_KINTO_INI, 'init',
+                  '--backend', 'memory'])
+            mocked_logging.basicConfig.assert_called_with(
+                format=DEFAULT_LOG_FORMAT)
```

Comment:
I don't think this test is necessary, `info` by default is fine no? logging default is warning btw